### PR TITLE
Feat/bridge axelar scripts

### PR DESCRIPTION
**Pull request by giovannifranchi**

### CODE REVIEW

File: script/Blast/upgrade/01_configureTokenBridgeFacetBlast.s.sol [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+/// @dev deploys the TokenBridge facet and cuts it onto the TokenProxy diamond
+contract ConfigureTokenBridgeFacet is Script {
+    function run() public {
+        uint256 deployerPrivateKey = vm.envUint("DEPLOYER_KEY");
+        address gateway = vm.envAddress("GATEWAYWAY_ADDRESS");
+        address gasService = vm.envAddress("GAS_SERVICE_ADDRESS");
+        address payable tokenProxyAddress = payable(
+            vm.envAddress("TOKEN_PROXY_ADDRESS")
+        );
+        address multiSigWalletAddress = vm.envAddress(
```
Comment (0xinsi) [inline script/Blast/upgrade/01_configureTokenBridgeFacetBlast.s.sol]: This is no long used anywhere.

File: script/Blast/upgrade/01_configureTokenBridgeFacetBlast.s.sol [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+            "MULTISIG_WALLET_ADDRESS"
+        );
+
+        vm.startBroadcast(deployerPrivateKey);
+
+        // deploy TokenBridge facet
+        ITokenBridge tokenBridgeFacet = new TokenBridge(gateway, gasService);
+        console.log("TokenBridge Facet Address: ", address(tokenBridgeFacet));
+
+        // getfacet cuts from TokenBridge facet
```
Comment (0xinsi) [inline script/Blast/upgrade/01_configureTokenBridgeFacetBlast.s.sol:28]:
```suggestion
        // get facet cuts from TokenBridge facet
```

File: script/Arbitrum/upgrade/upgrade-token-bridge-arbitrum.sh [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+# Get ETH balance in Wei
+DEPLOYER_BALANCE_DEC=$(cast balance $DEPLOYER_ADDRESS --rpc-url $RPC_URL)
+
+# Convert from Wei to Ether
+DEPLOYER_BALANCE_ETH=$(cast from-wei $DEPLOYER_BALANCE_DEC)
+echo -e "Deployer address balance is $DEPLOYER_BALANCE_ETH ETH.\n"
+
+# Run forge scripts
+forge script script/Blast/upgrade/${UPGRADE_SCRIPT} --rpc-url $RPC_URL --verify --broadcast --verifier-url $VERIFIER_URL
```
Comment (0xinsi) [inline script/Arbitrum/upgrade/upgrade-token-bridge-arbitrum.sh]: The path needs to be made for Arbitrum

File: script/Arbitrum/upgrade/01_configureTokenBridgeFacetArbitrum.s.sol [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+/// @dev deploys the TokenBridge facet and cuts it onto the TokenProxy diamond
+contract ConfigureTokenBridgeFacet is Script {
+    function run() public {
+        uint256 deployerPrivateKey = vm.envUint("DEPLOYER_KEY");
+        address gateway = vm.envAddress("GATEWAYWAY_ADDRESS");
+        address gasService = vm.envAddress("GAS_SERVICE_ADDRESS");
+        address payable tokenProxyAddress = payable(
+            vm.envAddress("TOKEN_PROXY_ADDRESS")
+        );
+        address multiSigWalletAddress = vm.envAddress(
```
Comment (0xinsi) [inline script/Arbitrum/upgrade/01_configureTokenBridgeFacetArbitrum.s.sol]: No longer used.

File: script/Arbitrum/upgrade/01_configureTokenBridgeFacetArbitrum.s.sol [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+            "MULTISIG_WALLET_ADDRESS"
+        );
+
+        vm.startBroadcast(deployerPrivateKey);
+
+        // deploy TokenBridge facet
+        ITokenBridge tokenBridgeFacet = new TokenBridge(gateway, gasService);
+        console.log("TokenBridge Facet Address: ", address(tokenBridgeFacet));
+
+        // getfacet cuts from TokenBridge facet
```
Comment (0xinsi) [inline script/Arbitrum/upgrade/01_configureTokenBridgeFacetArbitrum.s.sol:28]:
```suggestion
        // get facet cuts from TokenBridge facet
```

**Review by 0xinsi — CHANGES_REQUESTED** [review]
Minor changes but otherwise LGTM!

File: script/Blast/upgrade/01_configureTokenBridgeFacetBlast.s.sol (hunk 1 above)
Comment (giovannifranchi) [reply script/Blast/upgrade/01_configureTokenBridgeFacetBlast.s.sol]: fixed here(caff255801eae8633e349f9a1e048f780be8872d)

File: script/Arbitrum/upgrade/01_configureTokenBridgeFacetArbitrum.s.sol (hunk 4 above)
Comment (giovannifranchi) [reply script/Arbitrum/upgrade/01_configureTokenBridgeFacetArbitrum.s.sol]: fixed here(caff255801eae8633e349f9a1e048f780be8872d)

File: script/Arbitrum/upgrade/upgrade-token-bridge-arbitrum.sh (hunk 3 above)
Comment (giovannifranchi) [reply script/Arbitrum/upgrade/upgrade-token-bridge-arbitrum.sh]: fixed here(caff255801eae8633e349f9a1e048f780be8872d)

**Pull request merged**